src/simplicial-hott/04-extension-types.rzk.md

```diff
@@ -62,8 +62,10 @@ This is a literate `rzk` file:
   ( X : ψ -> ζ -> U)
   ( f : { (t , s) : I * J | (ϕ t /\ ζ s) \/ (ψ t /\ χ s)} -> X t s )
   : Equiv
-    ( {t : I | ψ t} -> ({ s : J | ζ s} -> X t s [ χ s |-> f (t , s) ]) [ ϕ t |-> \{s : J | ζ s} -> f (t , s)])
-    ( { (t , s) : I * J | ψ t /\ ζ s} -> X t s [(ϕ t /\ ζ s) \/ (ψ t /\ χ s) |-> f (t , s)])
+    ( {t : I | ψ t} -> ({ s : J | ζ s} -> X t s [ χ s |-> f (t , s) ])
+      [ ϕ t |-> \{s : J | ζ s} -> f (t , s)])
```

Comment:
I'm not sure if this is decidedly better here, but remember that shapes are coerced to proper constrained cubes and lambda abstractions do not need to be annotated:

```suggestion
    ( (t : ψ) -> ((s : ζ) -> X t s [ χ s |-> f (t , s) ])
      [ ϕ t |-> \ s -> f (t , s)])
```